tests/router/common.py

```
@@ -2168,6 +2168,182 @@ async def send_progressive_requests():
+                    prefill_dp_rank is not None
+                ), "Missing prefill_dp_rank in response body: " + "\n".join(body)
+                return int(prefill_worker_id), int(prefill_dp_rank)
+
+        async def test_sync():
+            await wait_for_frontend_ready(
+                frontend_url=frontend_url,
+                expected_num_workers=decode_workers.num_workers,
+                timeout=120,
+            )
+
+            suffix = random.randint(100_000, 999_999)
```

Ruff (0.15.14)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)